test/suites/storage.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/sh
+
+configure_lvm_loop_device() {
+  lv_loop_file=$(mktemp -p "${TEST_DIR}" XXXX.lvm)
+  truncate -s 4G "${lv_loop_file}"
+  pvloopdev=$(losetup --show -f "${lv_loop_file}")
+  if [ ! -e "${pvloopdev}" ]; then
+    echo "failed to setup loop"
+    false
+  fi
+
+  pvcreate "${pvloopdev}"
+
+  # shellcheck disable=2039
+  local  __tmp1="${1}"
+  # shellcheck disable=2039
+  local  res1="${lv_loop_file}"
+  if [ "${__tmp1}" ]; then
+      eval "${__tmp1}='${res1}'"
+  else
+      echo "${res1}"
+  fi
+
+  # shellcheck disable=2039
+  local  __tmp2="${2}"
+  # shellcheck disable=2039
+  local  res2="${pvloopdev}"
+  if [ "${__tmp2}" ]; then
+      eval "${__tmp2}='${res2}'"
+  else
+      echo "${res2}"
+  fi
```

Comment:
The BLA=$(fun) isn't used, so can we drop that part of the code and add a comment explaining the other mode?